back-end/api/controllers/projects.controller.js

```diff
@@ -102,22 +106,23 @@ export default class ProjectsController {
     }
 
 
-
     /**
      * A GET API for getting a project object with a particular id from MongoDB.
+     * @param req
+     * @param res
+     * @param next
+     * @returns {Promise<void>}
      */
     static async apiGetProjectByID(req, res, next) {
-        try {
-            let id = req.params.id || {};
-            let project = await ProjectsDAO.getProjectByID(id);
-            if (!project) {
-                res.status(404).json({ error: "Not found" });
+        let id = req.params.id || {};
+        const getprojectByIDResponse = await ProjectsService.getProjectbyID(id);
+        switch (getprojectByIDResponse.status) {
```

Comment:
Nice call on switching to a switch statement since we now have a services layer that only returns "success"/"failure".